src/routes/info/[slug]/+page.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  export let data;
+  import { goto } from "$app/navigation";
+  import MarkdownBody from "$lib/components/MarkdownBody.svelte";
+</script>
+
+{#if data && data.accessPolicies.includes(`markdowns:${data.slug}:update`)}
```

Comment:
Should use apiNames instead of constant string, see BUCKET as an example for dynamic access policy names